doc/api/modules.md

```diff
@@ -491,8 +491,8 @@ added: v0.0.1
 
 * {string}
 
-The file name of the current module. This is the resolved absolute path of the
-current module file.
+The file name of the current module. This is current module file's absolute path
```

Comment:
Just an idea, but I think `file` could be omitted or moved before `path`, which might be even clearer?

Reply:
So either `current module's absolute path` or else `current module's absolute file path`? It's subtle, but I wonder if that might make someone expect `__dirname` value instead of `__filename` if a module is loaded or invoked at the directory level (e.g., `node .` instead of `node index.js`).